gems/aws-sdk-core/lib/aws-sdk-core/process_credentials.rb

```diff
@@ -0,0 +1,54 @@
+module Aws
+  class ProcessCredentials
+
+    include CredentialProvider
+    include RefreshingCredentials
+
+    def initialize(process)
```

Comment:
could you add more documentation for `#initialize` like what kind of process variable it's expecting?